src/v/ssx/metrics.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2022 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "config/configuration.h"
+#include "utils/hdr_hist.h"
+
+#include <seastar/core/metrics.hh>
+#include <seastar/util/bool_class.hh>
+
+namespace ssx {
```

Comment:
```suggestion
namespace ssx::metrics {

Reply:
I've reverted to not using the helper so this is now outdated.